flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdSelectivity.scala

```diff
@@ -256,6 +257,13 @@ class FlinkRelMdSelectivity private extends MetadataHandler[BuiltInMetadata.Sele
     }
   }
 
+  def getSelectivity(
+      rel: CommonPhysicalWindowTableFunction,
+      mq: RelMetadataQuery,
+      predicate: RexNode): JDouble = {
+    estimateSelectivity(rel, mq, predicate)
+  }
+
```

Comment:
Although this method `getSelectivity(rel: RelNode, mq: RelMetadataQuery, predicate: RexNode)` is only effective for `BatchPhysicalRel`, I think most of the methods are actually effective for stream operators. Can we remove this limitation.

Reply:
For this particular case, what we're missing is the identification of the new node `WindowTableFunction`, and it's probably not a good time to modify the default processing logic as to whether the exact same default processing as the batch is used on the stream, WDYT?

Reply:
> For this particular case, what we're missing is the identification of the new node `WindowTableFunction`, and it's probably not a good time to modify the default processing logic as to whether the exact same default processing as the batch is used on the stream, WDYT?

I got it. LGTM+1